testconf/imqsauth.json

```
@@ -0,0 +1,18 @@
+{
+	"Authaus": {
+		"Log": {
+			"Filename": ".\\log\\imqsauth.log"
+		},
+		"HTTP": {
+			"Bind": "0.0.0.0",
+			"Port": "2003"
+		},
+		"DB": {
+			"Driver": "postgres",
+			"Host": "127.0.0.1",
+			"Database": "auth",
```

```suggestion
			"Database": "unittest_auth",
```
Or just `test_auth`.